driver-examples/src/main/java/com/datastax/driver/examples/apollo/AstraReadCassandraVersion.java

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package com.datastax.driver.examples.apollo;
+package com.datastax.driver.examples.astra;
```

Comment:
Hmm there is a compilation issue here, the package was renamed to `com.datastax.driver.examples.astra` but the directory is still `com/datastax/driver/examples/apollo` -> should be `com/datastax/driver/examples/astra`.

Reply:
ah, missed it. Resolved now